### PR TITLE
Made some small changes.

### PR DESCRIPTION
**Pull request by c0nfigurati0n**
- Redacted a lot more of the emails, etc on the screenshots.
- Updated the read me file.
- Updated some other stuff as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,3 +39,7 @@ Please take a look at existing pages to make sure new page follow the same struc
 
 4) Create a pull request for the new page.
 
+# Copyright Notice
+All contect in this repository is copyright ® of SimpleLogin SAS.
+
+**Copyright ® 2019 - 2023 SimpleLogin SAS. All rights reserved.**
```
Comment (nguyenkims) [inline README.md:45]: I wouldn't put the year here as we might forget to update it in 2 years :).

**Review by nguyenkims — COMMENTED** [review]
Thanks for making the PR! I think greying elements using a black box doesn't look good visually. Hiding the date or even the alias address isn't needed as these screenshots are made on a demo account and we often delete an alias after it appears in a screenshot anw. So I'd recommend to stay with the current images.

**Comment by therealmrmuffin** [comment]
>I think greying elements using a black box doesn't look good visually.

I worked with what i had lol. Sorry it didn't turn out to your liking :(

> Hiding the date or even the alias address isn't needed as these screenshots are made on a demo account and we often delete an alias after it appears in a screenshot.

I thought so, but what i have learned is "do not give any attacker potential information they can use to 'hack' you".

> So I'd recommend to stay with the current images.

It's your call.

File: README.md (hunk 1 above)
Comment (therealmrmuffin) [reply README.md:45]: Could be done. Altho, i don't now if it will legally still be valid then.

Comment (nguyenkims) [reply README.md:45]: I think the documentation doesn't really need a copyright so that should be ok.

**Comment by nguyenkims** [comment]
> > I think greying elements using a black box doesn't look good visually.
> 
> I worked with what i had lol. Sorry it didn't turn out to your liking :(
> 

All screenshots are made using the same software until now, I think having a consistent style is always nicer :)

> > Hiding the date or even the alias address isn't needed as these screenshots are made on a demo account and we often delete an alias after it appears in a screenshot.
> 
> I thought so, but what i have learned is "do not give any attacker potential information they can use to 'hack' you".
> 

I wouldn't be worried about that as our demo account is also a good example of how SL can protect a mailbox from unwanted spams.

> > So I'd recommend to stay with the current images.
> 
> It's your call.

Let's keep the current image then :)

**Comment by therealmrmuffin** [comment]
>All screenshots are made using the same software until now, I think having a consistent style is always nicer :)

Mind telling me what software?

> I wouldn't be worried about that as our demo account is also a good example of how SimpleLoign can protect a mailbox from unwanted spams.

If your really not a all worried. Then you can close this PR if you want.

> Let's keep the current image then :)

Will do. 👍

**Comment by nguyenkims** [comment]
@therealmrmuffin We're using mac and either use https://shottr.cc/?utm_source=pocket_mylist or https://cleanshot.com/?utm_source=pocket_mylist

**Comment by nguyenkims** [comment]
> Ps i would like to point out that this image does expose your email tho https://raw.githubusercontent.com/simple-login/documentation/master/docs/mailbox/new-mailbox.png.

This isn't a real mailbox :), its address is totally fictive.

**Comment by therealmrmuffin** [comment]
> @therealmrmuffin We're using mac and either use https://shottr.cc/?utm_source=pocket_mylist or https://cleanshot.com/?utm_source=pocket_mylist

@nguyenkims it doesn't look like it from the fact that you have redacted the email in the first place. But if that's the case, then i guess it indeed doesn't matter too much. Do you want to case to PR? Seen as it doesn't bring anything of value to SimpleLogin.

**Pull request closed without merging**